portals/application/configs/klear/TerminalModelsList.yaml

```diff
@@ -127,7 +127,7 @@ production:
       labelOnEdit: true
       action: run-generic-code
       class: ui-silk-control-play-blue
-    runSpecificCode_dialog:
+    runSpecificCode_dialog: &runSpecificCode_dialog
```

Comment:
> We usually call YAML links ending with Link word
Ok, I'll replace some other dialogs names as well
> where is this link used?
Nowhere, although every single dialog declaration is referable.